client/store/trailsStore_copy.js

```diff
@@ -19,16 +18,17 @@ export const fetchTrails = createAsyncThunk(
     if (lon) {
       params += `&lon=${lon}`;
     }
-    const radius = 500;
```

Comment:
Why no radius ?

Reply:
I was getting more trails result without the radius than with the radius do you think it’s okay if Trails result don’t always come in for every location?